bin/workflows/pycbc_create_offline_search_workflow

```diff
@@ -213,6 +212,49 @@ statfiles = []
 statfiles += wf.setup_trigger_fitting(workflow, insps, hdfbank,
                                       final_veto_file, final_veto_name)
 
+# setup the detector combination coincs for the filtering jobs
+full_insps = insps
+ctags = [tag, 'full']
+bg_files = []
+final_bg_files = []
+ifo_ids = {}
+
+# Get the ifo precedent list for the multiifo coinc jobs
+ifo_precedence_list = workflow.cp.get_opt_tags('workflow-coincidence', 'timeslide-precedence', ctags)
+for ifo, file in zip(*insps.categorize_by_attr('ifo')):
+    ifo_ids[ifo] = ifo_precedence_list.index(ifo)
+
+# Generate the possible detector combinations from 2 detectors
+# to the number of trigger files.
+for i in range(2, len(insps)+1):
+    combinations = itertools.combinations(ifo_ids.keys(), i)
+    for ifocomb in combinations:
+        combination_prec = {}
+        inspcomb = wf.core.FileList()
+        # create the precedence list for the detector combination
+        for ifo in ifocomb:
+            combination_prec[ifo] = ifo_ids[ifo]
+
+        # Use the inspiral files for the detector combination only.
+        for ifo, file in zip(*insps.categorize_by_attr('ifo')):
+            if ifo in combination_prec:
+                inspcomb += file
+
+        # Determine the pivot-ifo and fixed-ifo based on the precedent list
+        pivot_ifo = min(combination_prec, key=combination_prec.get)
+        del combination_prec[pivot_ifo]
+        fixed_ifo = min(combination_prec, key=combination_prec.get)
```

Comment:
comment here: `fixed_ifo is assigned to be the ifo with the next smallest index value`